test/issues.jl

```diff
@@ -46,4 +46,12 @@
         @test p.plots[1].plots[1].transformation.model[][3, 4] == 10.0
         @test p.plots[1].plots[2].transformation.model[][3, 4] == 10.0
     end
+
+    @testset "#4883 colorscale that breaks sort" begin
+        data = Float32.(Makie.peaks())
+        f, a, p = image(data; colorscale = -)
```

Comment:
Did that error without the Float32?

Reply:
It did, because I was comparing the extrema in Float64 to the extrema in Float32.  

```julia-repl
julia> mp = maximum(Makie.peaks())
8.075173545159231

julia> mpf32 = Float32(mp)
8.075173f0

julia> mp == mpf32
false
```

Reply:
Ah so it's just for the tests below and not because of some internal problem

Reply:
Yeah, I guess in this case Julia type promotes both numbers to Float64, so the lossy conversion kicks in.  And I don't want to test approximately here.